backend/unitTests/searchItemTest.py

```
@@ -0,0 +1,161 @@
+        if response.status_code == 200:
+            result = response.json()
+            if result.get("success"):
+                results = result.get("results", [])
+                print(f"Found {len(results)} similar items")
+                for i, item in enumerate(results):
+                    print(f"  {i+1}. {item.get('name')} (Score: {item.get('score', 'N/A')})")
+                return True
+            else:
+                print(f"Search failed: {result.get('error')}")
+                return False
+        return False
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)